.github/workflows/core.yml

```
@@ -697,16 +697,18 @@ jobs:
 
   core_ui_comment:
     name: Post comment with UI diff URLs
-    if: github.event_name == 'pull_request' && github.event.pull_request.head.repo.full_name == github.repository
+    if: github.event.pull_request.head.repo.full_name == github.repository
```

IIUC `github.event.pull_request.head.repo.full_name == github.repository` will be false on non-PR events, causing the job to be skipped -> https://github.com/trezor/trezor-firmware/actions/runs/16385769648/job/46305061695
```suggestion
    # skip UI comment job on external PRs (see #5381)
    if: !(github.event_name == 'pull_request' && github.event.pull_request.head.repo.full_name != github.repository)
```